definitions/tools/picard_merge_vcfs.cwl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Picard MergeVcfs"
+baseCommand: ["/usr/bin/java", "-jar", "/opt/picard/picard.jar", "MergeVcfs"]
+requirements:
+    - class: ResourceRequirement
+      ramMin: 40000
+    - class: InlineJavascriptRequirement
+    - class: DockerRequirement
+      dockerPull: "mgibio/picard-cwl:2.18.1"
+arguments:
+    ["O=merged.vcf.gz"]
```

Comment:
@tmooney Do you have a suggestion for the name?

Reply:
Something like how this tool does it is what I had in mind:
https://github.com/genome/analysis-workflows/blob/9daeb6b8856498d5b276a996d4be1f9582216f2e/definitions/tools/select_variants.cwl#L40-L42

Reply:
Oh I see, instead of having the arguments: block. This would allow us to control the output name with a parameter then?

Thanks! I just wanted to know for reference and improvement.